_posts/2015-03-03-Polls-API-Blueprint-Tutorial-Examples.md

```diff
@@ -0,0 +1,87 @@
+---
+title: New API Blueprint tutorial
+excerpt: Learning API Blueprint has never been easier.
+layout: post
+date: 2015-03-25 16:00:00 +0000
+author: zdenek
+published: true
+comments: true
+---
+
+Everything at Apiary starts with API Blueprint and API Blueprint starts with its tutorial. Since providing superb and concise documentation is our priority, there is no other document that should stand more for our values.
+But frankly, the old tutorial was too long, unnecessary complex and wasn't complemented by blueprint examples, real code and the rest of Apiary. This is changing right now with our new API Blueprint tutorial.
+
+We have completely rewritten the tutorial, stripped any clutter, and saved advanced topics for later.  In addition, the new tutorial no longer relies on any upfront knowledge of Markdown. You can get started with API Blueprint in less than thousands words, counting blueprint snippets and footnotes!
```

Comment:
'As a result, we decided to completely rewrite the tutorial so that any user, despite their experience, could easily use and understand API Blueprint without any prior knowledge of Markdown.'

-- I have little idea what you mean with some of the above examples 'get started in less than a thousand words..'.. Would any beginner understand that?